packages/client/src/pages/PortalSettings/categories/developer-tools/Webhooks/WebhookHistory/sub-components/WebhookHistoryTable/HistoryTableView/HistoryTableRow.js

```diff
@@ -17,6 +17,8 @@ import RetryIcon from "PUBLIC_DIR/images/refresh.react.svg?url";
 import InfoIcon from "PUBLIC_DIR/images/info.outline.react.svg?url";
 
 import { useTranslation } from "react-i18next";
+import { getCookie } from "../../../../../../../../../../../common/utils";
```

Comment:
Use workspaces link instead of long path

import { getCookie } from "@docspace/common/utils";